src/Raven.Client/Documents/Session/IRevisionsSessionOperations.cs

```diff
@@ -74,5 +74,22 @@ public interface IRevisionsSessionOperations
         /// Returns the number of revisions for specified document.
         /// </summary>
         long GetCountFor(string id);
+        
+        /// <summary>
+        /// Access the lazy revisions operations 
+        /// </summary>
+        ILazyRevisionsOperations Lazily { get;}
+    }
+    /// <summary>
+    /// Revisions advanced synchronous Lazy session operations
+    /// </summary>
+    public interface ILazyRevisionsOperations
+    {
+        Lazy<List<T>> GetFor<T>(string id, int start = 0, int pageSize = 25);
```

Comment:
can you copy the comments IRevisionsSessionOperations here? Remember to adjust the text there if needed.

Reply:
done